third_party/challenge_bypass_ristretto/Cargo.toml

```diff
@@ -15,7 +15,7 @@ crate-type = [
     "rlib",
 ]
 name = "challenge_bypass_ristretto_ffi"
-path = "../../vendor/challenge_bypass_ristretto_ffi/src/lib.rs"
+path = "../../third_party/challenge_bypass_ristretto/src/lib.rs"
```

Comment:
You can remove the `path` directive altogether now that the source is local. `src/lib.rs` is the default.

Reply:
Fixed